modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -450,15 +451,25 @@ export default {
           if (docData.type !== this.docType) {
             this.docType = docData.type;
           }
+
           this.original = klona(docData);
-          this.docFields.data = docData;
+          this.docFields.data = this.schema.reduce(addDefaultValues, docData);
```

Comment:
add the default values defined in the schema fields to fix conditional fields not showing up when editing existing docs, relying on new fields that were not saved in the document